products/kms/terraform.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  KeyRing: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+
+      ~> **Note:** KeyRings cannot be deleted from Google Cloud Platform.
+      Destroying a Terraform-managed KeyRing will remove it from state but
+      *will not delete the resource on the server.*
+    id_format: "projects/{{project}}/locations/{{location}}/keyRings/{{name}}"
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "kms_key_ring_basic"
+        primary_resource_id: "example-keyring"
+        skip_test: true
```

Comment:
Can you add a comment about how the tests for this resource need extra stuff and so the generated test is not sufficient?